Security Headset panic button

DrDiasyl opened the pull request:
# **DESIGN DOCUMENT - https://hackmd.io/@9rEOsZB2S9y6Cx12KRCAAg/BkB9J79L2** Read it, really
## About The Pull Request
Adds a Panic Button to the Security bowman (not normal ones) and HoS headsets.
This button sends a message into the Security channel that the wearer needs backup (while not providing any other information), and announces to everyone around the Officer that he called in backup (with an obvious sound)

![image](https://github.com/tgstation/tgstation/assets/42353186/bb3a2b81-e0b1-4bc9-9705-cac92231eafa)
![image](https://github.com/tgstation/tgstation/assets/42353186/1fef459b-d233-4d02-8375-8f13f2f8503e)

**The Panic Button doesn't work if the wearer is incapacitated, dead, or the headset is EMP'd.**
## Why It's Good For The Game
After reading comments in #66578 and why it was closed, I understood that because it was overly detailed. You could've reported much information to the Security channel in a few clicks, discouraging talking. As such, many people there suggested this PR kind of feature.

The message of the Panic Button is enough to understand that this person is in danger, but not enough to understand where he is or what kind of danger he is facing. Leaving the job of locating the said Officer and sending in back-up to the Warden, HoS, or AI, encouraging teamwork in the Security department. (Read the design document)

The calling for backup is obvious to everyone who has Officer in sight, so the antagonist understands that *probably* more Security Officers are on their way and that they must run away (or remove sensors) ASAP.

*Sometimes less is better than more...*
## Changelog
:cl:
add: Add panic buttons to the Security bowman and HoS headsets. When pressed, they notify Security that the wearer is currently in distress and requires backup, while not giving any details like the position, leaving the job of location to the Warden, HoS, or AI.
/:cl:

DrDiasyl committed:
panic button in sec headsets

carlarctg commented:
HELL no, this is just NOT a good idea in my honest opinion, we don't need a button to instantly alert sec over comms, the ability to kill people before they can scream about it in comms is a vital part of antagonist gameplay and this completely screws with it when literally all a sec officer needs to do the same is '.h HLEP'

This will worsen helicopter sec and make it a LOT harder for antags to get things done when the majority of sec is on point. 
Also, this fucks over mute toxin hard, and there are definitely going to be plenty of random crewmembers running around with sec headsets that they powergamed out of a locker (alongside their stun baton) as well, which will also screw you over as an antagonist because they'll just be tracked through the crew monitor after calling it out

HOWEVER! If this was changed to be a SecGear vendor premium item with a reasonably expensive cost, I would be in favor. It'd be a nice mid-round progression thing for security.

Have you made sure EMPs break it by the way?

DrDiasyl commented:
> HELL no, this is just NOT a good idea in my honest opinion, we don't need a button to instantly alert sec over comms, the ability to kill people before they can scream about it in comms is a vital part of antagonist gameplay and this completely screws with it when literally all a sec officer needs to do the same is '.h HLEP'
> 
> This will worsen helicopter sec and make it a LOT harder for antags to get things done when the majority of sec is on point. Also, this fucks over mute toxin hard, and there are definitely going to be plenty of random crewmembers running around with sec headsets that they powergamed out of a locker (alongside their stun baton) as well, which will also screw you over as an antagonist because they'll just be tracked through the crew monitor after calling it out
> 
> HOWEVER! If this was changed to be a SecGear vendor premium item with a reasonably expensive cost, I would be in favor. It'd be a nice mid-round progression thing for security.
> 
> Have you made sure EMPs break it by the way?

It in fact doesn't work when EMP'd. Also, it doesn't work when you're paralyzed or stunned (aka anything that blocks interactions)

About your other concerns, if Security officers really want to they can have the "help" message copypasted, have a remote signaller on them, or just always walk around with a second Officer. Security Officers should not be the main targets of antagonists, they must be the barriers between the antagonist and his victory, it's kinda funny that right now they're treated like gear pinatas.
People who powergame headsets, that's a rule violation and should not happen, those who can steal these are usually (you guess it) antags!
And crew sensors are not always tracked, it will take *some time* for Warden to see where the hell you are.

I think making it a SecVend gear is actually worse, I put it into a headset so if you remove it you eliminate his communications ENTIRELY! If you make this a separate gear and remove the headset, he still will be able to call-in for help.

DrDiasyl commented:
Also, those powergamers usually tide lockers at the outposts, don't have batons, and have no idea why you mentioned those.

Helg2 commented:
> People who powergame headsets, that's a rule violation and should not happen, those who can steal these are usually (you guess it) antags!

Did you try playing lrp?

DrDiasyl commented:
> > People who powergame headsets, that's a rule violation and should not happen, those who can steal these are usually (you guess it) antags!
> 
> Did you try playing lrp?

I did, I have no idea how they're doing that if this is an obvious rule violation. Hence, this is an admin and not coder issue, and should not be considered there.
We should stop using coders instead of admins to stop rule violations.

Singul0 commented:
> About your other concerns, if Security officers really want to they can have the "help" message copypasted, have a remote signaller on them, or just always walk around with a second Officer.

Problem is, with all the examples you list out none of them are neither incentivized nor clear hell, One of them is straight up frowned upon. Using hotmic, nanites and remote signallers are rewards for knowing the mechanics that surround them not just handed out to them for free free for them to use at the slightest trigger of their paranoia.

>Security Officers should not be the main targets of antagonists

Except they are, Heretics always have ond sec member as their sacrifice targets and BB's, Clings, Traitors all have them if they roll unlucky with their objectives.

Helg2 commented:
Cult usually also has the sec member as a target.

DrDiasyl commented:
> > About your other concerns, if Security officers really want to they can have the "help" message copypasted, have a remote signaller on them, or just always walk around with a second Officer.
> 
> Problem is, with all the examples you list out none of them are neither incentivized nor clear hell, One of them is straight up frowned upon. Using hotmic, nanites and remote signallers are rewards for knowing the mechanics that surround them not just handed out to them for free free for them to use at the slightest trigger of their paranoia. (Sec mains, in my experience is one of the most paranoid of any other mains by a wide margin)
> 
> > Security Officers should not be the main targets of antagonists
> 
> Except they are, Heretics always have one sec member as their sacrifice targets and BB's, Clings, Traitors all have them if they roll unlucky with their objectives.

Locking calling in for help by the knowledge level is silly, Security needs to be able to call in for help and work as a team to solve station threats (while also not being under threat of dying when typing in a message, I also recommend reading the design doc). And only Security by default can call in for help that easily, normal people don't get sec headsets, and those who do are usually malicious.
Also which one is frowned upon?

Heretics? Alright, sure. Others? This doesn't make them the most main target, as those objectives only roll when as you said they're "unlucky". And Security itself still exists to be the opposing force of antagonists.

DrDiasyl commented:
Also, want to note again, this is improved (very simplified) #66578, as much as to make it work more balanced considering the comments I read on it.

Singul0 commented:
> Locking calling in for help by the knowledge level is silly

Normally I would agree to this defence but it's not like it's a round changing/core mechanic. You can call for help anytime  if you know how the radio works. It doesnt take much skill to type out "HELP" before fighting the antag. It also heavily fucks over mute toxin users (does radio jammer work?). Why use things that need a long time to plan but allows for a stealth kill when all the seccie gotta do is press a single button.

DrDiasyl commented:
> > Locking calling in for help by the knowledge level is silly
> 
> Normally I would agree to this defence but it's not like it's a round changing/core mechanic. You can call for help anytime  if you know how the radio works. It doesnt take much skill to type out "HELP" before fighting the antag. It also heavily fucks over mute toxin users (does radio jammer work?). Why use things that need a long time to plan but allows for a stealth kill when all the seccie gotta do is press a single button.

First of all, radio jammer does work.
Second, if you're really going for a stealth kill you will probably disable the sensors of the Seccie as fast as you can.
Looking up the sensors take up sometime, and you're still not 100% sure if the person is in the last location they've been in, you can remove jumpsuit and just drag off the guy to anywhere else. 
Sensors are only saying area, Security already has bad time understanding in which part of the Greater Fore Port Maintenance the Captain is dying, or what the fuck is Fore and Port

Singul0 commented:
> > > Locking calling in for help by the knowledge level is silly
> > 
> > 
> > Normally I would agree to this defence but it's not like it's a round changing/core mechanic. You can call for help anytime  if you know how the radio works. It doesnt take much skill to type out "HELP" before fighting the antag. It also heavily fucks over mute toxin users (does radio jammer work?). Why use things that need a long time to plan but allows for a stealth kill when all the seccie gotta do is press a single button.
> 
> First of all, radio jammer does work. Second, if you're really going for a stealth kill you will probably disable the sensors of the Seccie as fast as you can. Looking up the sensors take up sometime, and you're still not 100% sure if the person is in the last location they've been in, you can remove jumpsuit and just drag off the guy to anywhere else. Sensors are only saying area, Security already has bad time understanding in which part of the Greater Fore Port Maintenance the Captain is dying, or what the fuck is Fore and Port
> 
> (Reminder that panic button doesn't give location, but just the fact that guy needs help)

It's not really a stealth kill if everyone knows some sec officer just pressed the button and promptly vanished off sensors after taking heavy damage. Also it's not that easy to turn off suit sensors they sould have to strip/emp them to disable it, Not to mention to kill them first.

Most warden office has a crew monitoring console right beside theif desk. A warden can just leave that tab open (or an AI for that matter) and look at the officer location to immediately tell sec their location. At least with before this change you would have to be attentive and carefully monitor the console. Medical personnel informing sec would also need time to transmit the information through appropiate channels which would take time.

DrDiasyl commented:
> > > > Locking calling in for help by the knowledge level is silly
> > > 
> > > 
> > > Normally I would agree to this defence but it's not like it's a round changing/core mechanic. You can call for help anytime  if you know how the radio works. It doesnt take much skill to type out "HELP" before fighting the antag. It also heavily fucks over mute toxin users (does radio jammer work?). Why use things that need a long time to plan but allows for a stealth kill when all the seccie gotta do is press a single button.
> > 
> > First of all, radio jammer does work. Second, if you're really going for a stealth kill you will probably disable the sensors of the Seccie as fast as you can. Looking up the sensors take up sometime, and you're still not 100% sure if the person is in the last location they've been in, you can remove jumpsuit and just drag off the guy to anywhere else. Sensors are only saying area, Security already has bad time understanding in which part of the Greater Fore Port Maintenance the Captain is dying, or what the fuck is Fore and Port
> > 
> > (Reminder that panic button doesn't give location, but just the fact that guy needs help)
> 
> It's not really a stealth kill if everyone knows some sec officer just pressed the button and promptly vanished off sensors after taking heavy damage. Also it's not that easy to turn off suit sensors they sould have to strip/emp them to disable it, Not to mention to kill them first.
> 
> Most warden office has a crew monitoring console right beside their desk. A warden can just leave that tab open (or an AI for that matter) and look at the officer location to immediately tell sec their location. At least with before this change you would have to be attentive and carefully monitor the console. Medical personnel informing sec would also need time to transmit the information through appropiate channels which would take time.

With the level of security coordination and geography, even message from the Warden will take time to parse, enough time to remove sensors.

And dead of Officer will be noticed after *some* time anyway.

DrDiasyl commented:
Also about Heretic and Cultist always having 1 security target.
Those both have the "funny hand" which instantly paralyses you (aka you can't talk - into the radio, **or press the panic button**)

Singul0 commented:
> With the level of security coordination and geography, even message from the Warden will take time to parse, enough time to remove sensors.
> 
> And dead of Officer will be noticed after _some_ time anyway.

It's really not enough time. The location name is displayed to the right of their name. A warden could do that within seconds of the broadcast.

The keg word here is **time** an officer death would be noticed given enough time but at least currently it would give you enough time to turn off their suit sensors or impersonate them. I cannot stress enogh that the cling mute sting whose main purpose is for a quiet death would be rendered useless against seccies making impersonating them harder as it already is.

dendyd00m commented:
im in two minds about this thing because generally i don't like automated communication it feels lazy and antithetical to the point of roleplaying but also i've always been of the sincere belief that sec should be an obstacle that are best avoided by antags rather than directly confronted and steamrolled with pure l33t gamer skill

so i guess what im saying is my opinion is nothing

Singul0 commented:
I fear that this change would lock antags into more loud murderbone (read:boring) ways of murdering. Why try to be stealthy when you're going to be found out anyways?

DrDiasyl commented:
Panic button is easily broken by EMP, so you can just buy a EMP flashlight and use it (or a EMP screech).

If it will be found that antags are being **too** lazy making EMP to kill their **only** Sec target, maybe I will add an easier way to break the panic button, but not now.

DrDiasyl commented:
> im in two minds about this thing because generally i don't like automated communication it feels lazy and antithetical to the point of roleplaying but also i've always been of the sincere belief that sec should be an obstacle that are best avoided by antags rather than directly confronted and steamrolled with pure l33t gamer skill
> 
> so i guess what im saying is my opinion is nothing

It's something, also the problem is already addressed in the design doc and is solved.

JohnFulpWillard commented:
I like the idea and think we can make it work, but we should probably gatekeep it some more. 

2 ideas I had were to limit this to the bowman’s headset, so people who steal regular radio don’t get it, and removing the bowman from security closets and move them to the clothes vending machine like service radio does it.

SomeoneYouProbablyKnow commented:
This is practical, strong and something a competent security force should have which is precisely the issue. We should not be a sec vs antag game and this is just another change that's pushing us in that direction. 
Yes ambush combat is a big issue (and the fact that it's faster than human typing) but this is only making sec "the protagonists" of the station (more so than they already are in terms of mechanics and culture)

SomeoneYouProbablyKnow commented:
> I fear that this change would lock antags into more loud murderbone (read:boring) ways of murdering. Why try to be stealthy when you're going to be found out anyways?

Unrelated (and a bit counterintuitive given my other point) but given the choice between murderboning and maint camping i'd prefer boning as it actually gives the crew more agency, but that's it's own can of worms and asterisks

DrDiasyl commented:
> This is practical, strong and something a competent security force should have which is precisely the issue. We should not be a sec vs antag game and this is just another change that's pushing us in that direction. 
> Yes ambush combat is a big issue (and the fact that combat itself is so much faster than human typing) but this is only making sec "the protagonists" of the station (more so than they already are in terms of mechanics and culture)

How Security being able to call in for help rapidly make them protagonist's?

The PR itself doesn't do such, it's just handy (for more read design doc). I just have stated that I think Security is the barrier between Antagonists and their objectives.

DrDiasyl commented:
Or you want to say that at this point giving Security anything is bad? Which is silly.

DrDiasyl commented:
> I like the idea and think we can make it work, but we should probably gatekeep it some more. 
> 
> 2 ideas I had to limit this, is to make the ability only on the bowman’s headset, so people who steal regular radio don’t get it, and removing the bowman from security closets and move them to the clothes vending machine like service radio does it. 
> 
> We could also add a timer before the effect goes through, simulating the person typing but during that time they could be running away or fending their attacker off. This will give at least some more leeway to the attacker while making the item still useful. 

Outposts usually have security clothing vendors, so I would say it should be at sec gear vendor.

Timer which works even with you moving is great too.
I can do these both.

SomeoneYouProbablyKnow commented:
> How Security being able to call in for help rapidly make them protagonist's?

Because they are treated as such, with their metaprotection and license to hunt antags, yes letting them mitgate risks of going in to fight alone or hunt in maints is reinforcing that. It's not a coordination tool, it's not there to help sec keep the peace it's there to fuck over antags that sec is fighting, us (sec) vs them (valids)

> The PR itself doesn't do such, it's just handy (for more read design doc). I just have stated that I think Security is the barrier between Antagonists and their objectives.

No i read your reasoning, i just disagree that those are going to be the outcomes.

NotSoEpikName commented:
if a maintainer will merge this 
its gonna be a freaking hell for antags
just dont do it please it not a good adia

JohnFulpWillard commented:
> if a maintainer will merge this 
> 
> its gonna be a freaking hell for antags
> 
> just dont do it please it not a good adia 

![image](https://github.com/tgstation/tgstation/assets/53777086/9f14a3c9-77c7-417d-9dd5-869cd9186eb8)

SomeoneYouProbablyKnow commented:
> > if a maintainer will merge this
> > its gonna be a freaking hell for antags
> > just dont do it please it not a good adia
> 
> ![image](https://user-images.githubusercontent.com/53777086/243198284-9f14a3c9-77c7-417d-9dd5-869cd9186eb8.png)

Oh fuck that i don't wanna be on the same side as this, merge it

NotSoEpikName commented:
> > if a maintainer will merge this
> > its gonna be a freaking hell for antags
> > just dont do it please it not a good adia
> 
> ![image](https://user-images.githubusercontent.com/53777086/243198284-9f14a3c9-77c7-417d-9dd5-869cd9186eb8.png)

in fact my older acc had name nigerfucker so it not that bad

DrDiasyl commented:
The least chaotic GitHub discussion

Crushtoe commented:
I don't like this PR but I don't wanna agree with that other guy so merge this PR

Sealed101 commented:
still feel like it would work out better if it was an upgrade kit or something that would be available at some later point of the game (advanced sec gear research node? sec gear vendor?) rather than roundstart equipment. but it's also making bowmans more tedious to loot which is based


also god fucking damnit komrade

DrDiasyl committed:
Does the John Fulp Willard proposal

Removes the security bowman headset from the basic security lockers, into the security lockers at the brig.

The normal security lockers (at outposts) still have the basic headset that you can use to overhear Sec comms, and also the rest of the things (Like a helmet, which too protects from the flashbangs)

Also now panic button is only at the bowman headset, not normal ones.

Co-Authored-By: John Willard <53777086+JohnFulpWillard@users.noreply.github.com>

DrDiasyl commented:
> I like the idea and think we can make it work, but we should probably gatekeep it some more.
> 
> 2 ideas I had to limit this, is to make the ability only on the bowman’s headset, so people who steal regular radio don’t get it, and removing the bowman from security closets and move them to the clothes vending machine like service radio does it.
> 
> We could also add a timer before the effect goes through, simulating the person typing but during that time they could be running away or fending their attacker off. This will give at least some more leeway to the attacker while making the item still useful.

Okay I did the first proposal.

DrDiasyl committed:
it got unticked for some reason

carlarctg commented:
I agree with SYPK. This doesn't encourage cooperation or help security keep the peace, it helps them roam maint hallways alone, seeking antagonists, with a safe safety whistle that helps makes them 'the protagonists'. I would be fine with it if heads had it, but letting every mall cop get one for free is just unreasonable IMO and will do nothing but improve validhunting.

'Security should be avoided not robusted' doesn't really get improved when you give security a free backup button they can use if they feel in danger, as that makes them more confident in rushing down antagonists alone which ends up meaning you spend half the round dodging assholes in maintenance and can't get anything done.

ArcaneMusic commented on code/game/objects/items/devices/radio/radio.dm:
```diff
@@ -585,4 +592,23 @@
+/obj/item/radio/proc/panic(mob/user)
+
+	if(panic_time > world.time)
+		to_chat(user, span_warning("The panic button cooldown didn't pass yet!"))
```
```suggestion
		balloon_alert(user, "on cooldown!")
```
Better to go with balloon alerts on visible things like this, especially with the context that it'd be more combat focused, so they keep their attention to the middle of the screen.

DrDiasyl commented:
> I agree with SYPK. This doesn't encourage cooperation or help security keep the peace, it helps them roam maint hallways alone, seeking antagonists, with a safe safety whistle that helps makes them 'the protagonists'. I would be fine with it if heads had it, but letting every mall cop get one for free is just unreasonable IMO and will do nothing but improve validhunting.
> 
> 'Security should be avoided not robusted' doesn't really get improved when you give security a free backup button they can use if they feel in danger, as that makes them more confident in rushing down antagonists alone which ends up meaning you spend half the round dodging assholes in maintenance and can't get anything done.

I'm gonna surprise you, but security could've done this before just by typing.
Panic button makes it just more convenient, while also _having_ them to be teamworking (as it only says who needs help, without stating location) as it would need a involvement of Warden or HoS.

ATH1909 commented:
> Security bowman headset now only spawns in the Brig security lockers and not outposts one (to prevent abuse of panic button by crew members, and making them more limited to Sec only). There is still a basic one in those that you can use to overhear Sec comms as usual, for flashbang protection you can put on a helmet which is too in the basic sec lockers.

sec helmets do not protect your ears, so this is actually really bad for any antag who doesn't want to get owned by a flashbang or sonic powder mix from almost a full screen away with very little counterplay

if you REALLY want to add this, I'd suggest making it an upgrade you can apply to headsets that's dispensed by a vendor or something

DrDiasyl commented:
> > Security bowman headset now only spawns in the Brig security lockers and not outposts one (to prevent abuse of panic button by crew members, and making them more limited to Sec only). There is still a basic one in those that you can use to overhear Sec comms as usual, for flashbang protection you can put on a helmet which is too in the basic sec lockers.
> 
> sec helmets do not protect your ears, so this is actually really bad for any antag who doesn't want to get owned by a flashbang or sonic powder mix from almost a full screen away with very little counterplay
> 
> if you REALLY want to add this, I'd suggest making it an upgrade you can apply to headsets that's dispensed by a vendor or something

Nice suggestion, but I think if this will be disliked by some other maintainer, I will probably do the second John's proposal instead of first one.

TheVekter commented:
All I can really do here is echo that this is a bad idea. Sec already has it relatively easy in that they can just go ":s HELP" which does the exact same thing as this.

SomeoneYouProbablyKnow commented:
I mean you gotta at least admit that this is more than just a small thing sec could already be doing when you have to limit them to secure brig lockers. Can't have it be both ways

ArcaneMusic commented on code/game/objects/items/devices/radio/headset.dm:
```diff
@@ -253,14 +255,16 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/heads/hos
```
So I know you mentioned that helmets offer ear protection as part of the justification to moving away from making security bowman's the default headset, but helmets do not have the ear protection component. This should likely be handled before going forward.

DrDiasyl replied:
I just reverted that proposal, will do the second John proposal

TheVekter commented:
Also @DrDiasyl TBH this PR does nothing to explain why this is a good idea, only saying that it's somehow better than the last PR which was already not great so it's a low bar of an idea.

How is this good for the game?

DrDiasyl commented:
> Also @DrDiasyl TBH this PR does nothing to explain why this is a good idea, only saying that it's somehow better than the last PR which was already not great so it's a low bar of an idea.
> 
> How is this good for the game?

Read the design document.

DrDiasyl committed:
Revert "Does the John Fulp Willard proposal"

This reverts commit 4e9a90a5feaba511157e8b48e8b6b8cc446f98e3.

DrDiasyl commented:
Okay mad people, I won't do the first John proposal, I will do the second John proposal of making pressing button take time (but you still can move and stuff during it)

TheVekter commented:
> > Also @DrDiasyl TBH this PR does nothing to explain why this is a good idea, only saying that it's somehow better than the last PR which was already not great so it's a low bar of an idea.
> > How is this good for the game?
> 
> Read the design document.

Done, I still think it's a bad idea. It takes AI all of a few seconds to track someone on camera and unless they're deep in maintenance then you've already given security all the info they need.

The ONLY good thing this would do is encourage more antags to buy EMP grenades, but it would still be bad because it would effectively make them mandatory.

DrDiasyl commented:
> > > Also @DrDiasyl TBH this PR does nothing to explain why this is a good idea, only saying that it's somehow better than the last PR which was already not great so it's a low bar of an idea.
> > > How is this good for the game?
> > 
> > 
> > Read the design document.
> 
> Done, I still think it's a bad idea. It takes AI all of a few seconds to track someone on camera and unless they're deep in maintenance then you've already given security all the info they need.
> 
> The ONLY good thing this would do is encourage more antags to buy EMP grenades, but it would still be bad because it would effectively make them mandatory.

We should account that most of the time it probably will be the Warden who tracks the location, although AI can do that, I think its more busy opening doors.

And about EMP grenades, you should not care that much about the Security calling in for help if you're not attacking them but _defending_ them. As if you're defending, they actually always had time to speak into the headset, especially before engaging.
If you're attacking them, you should buy some EMP grenades. But this doesn't make them mandatory.

DrDiasyl committed:
cooldown balloon alert

SinfulBliss commented:
The issue is this incentivizes sec to just be silent batong NPCs that don’t RP or speak on sec comms and just hunt antags wordlessly, which is already sort of a thing on LRP. 

Buttons for comms are a mistake and stifle RP and meaningful interactions

DrDiasyl commented:
> The issue is this incentivizes sec to just be silent batong NPCs that don’t RP or speak on sec comms and just hunt antags wordlessly, which is already sort of a thing on LRP. 
> 
> Buttons for comms are a mistake and stifle RP and meaningful interactions 

A single button which sends a extremely vague message which means only one thing ("I need help!") cannot replace the entire human vocabulary for Security Officers (as stated in design document).
They have other things to ask, declare, and talk about on radio, and to do so they will _have to_ write their own messages.

optimumtact commented:
yeah this is something we are just not interested in.

DrDiasyl commented:
> yeah this is something we are just not interested in.

Damn it, well I guess now I should go and riot at my local grocery store because "Orange man bad" or something.

But wasn't there a initial interest in that #66578 PR? As this is kinda of a simplification of it based on the comments I read on it.

Pumpkinoe commented:
I really dislike the idea of it being so easy to just call out and specify a location with the press of a button - isn't it more interesting to make security officers type it out and call for it? It's more engaging and encourages things like security officers running to call out, or maybe sacrificing a moment in the middle of combat to call for help
Rather than an extremely safe and instant 'call for help' button
Same with #66578 that it just removes a bit of RP which is lame

DrDiasyl commented:
> I really dislike the idea of it being so easy to just call out and specify a location with the press of a button - isn't it more interesting to make security officers type it out and call for it? It's more engaging and encourages things like security officers running to call out, or maybe sacrificing a moment in the middle of combat to call for help
> Rather than an extremely safe and instant 'call for help' button
> Same with #66578 that it just removes a bit of RP which is lame

It doesn't specify location. The problem with #66578 was it giving too much information and having too much types of messages you could send.

ExcessiveUseOfCobblestone commented:
>But wasn't there a initial interest in that https://github.com/tgstation/tgstation/pull/66578 PR? As this is kinda of a simplification of it based on the comments I read on it.

Except the one that explains why it was closed?

The real question is do we really want an inbuilt security should always pre-copy ":S help" button? I dont think thats ideal, especially as a roundstart item.

DrDiasyl commented:
Well alright anyway, this was at least good to try. *Unless something spontaneous like another headcoder busting in and liking idea happens,* I will just accept Oranges man decision.

EricZilla commented:
I know this is a million years too late but a panic mod for headset via research or bepis would have been the better implementation, then also make it incompatible with bowman so you have to choose ear safety or panic assist
